# Fix CMake minimum version for clickhouse-cpp
# CMake 3.27+ deprecated versions < 3.5, CMake 4.0.0 removed compatibility
--- CMakeLists.txt.orig	2025-11-01 10:00:00.000000000 +0000
+++ CMakeLists.txt	2025-11-01 10:00:00.000000000 +0000
@@ -1,4 +1,4 @@
-CMAKE_MINIMUM_REQUIRED (VERSION 3.0.2)
+CMAKE_MINIMUM_REQUIRED (VERSION 3.5)

 INCLUDE (cmake/cpp17.cmake)
 INCLUDE (cmake/subdirs.cmake)
